Gemfile

```diff
@@ -65,12 +65,16 @@ gem "sidekiq"
 # Elasticsearch with chewy [https://github.com/toptal/chewy]
 gem "chewy"
 
+# Ruby implementation of GraphQL [https://github.com/rmosolgo/graphql-ruby]
+gem "graphql"
+
 group :development, :test do
   gem "bullet"
 
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[mri mingw x64_mingw]
   gem "factory_bot_rails"
+  gem "pry", "~> 0.14.2"
```

Comment:
I think we don't need pry, because we have `binding.irb`, wdyt?

Reply:
personal preference